ocaml/jane/doc/extensions/local/reference.md

```diff
@@ -460,11 +460,8 @@ value `x` remains available.
 
```

Comment:
The title of this section "local-returning functions" should be renamed to mention `exclave_`. I suggest:

## Use `exclave_` to return a stack allocation

Reply:
This also requires updating references to this section, e.g. "See "Local-returning functions" below."

Reply:
Yes I also thought about this, but thought maybe "local-returning functions" is what users want, and "exclave" is just a way (currently the only way) to achieve that. Let me know if you still think we shoud change to "exclave". (I have no preference).

Reply:
I think that the either the title or the first line of the section should mention “exclave” to help users scan the doc.

maybe the first sentence approach is enough?

> TL;DR: Use `exclave_` to return a local value.

“Local value” is better than “local allocation.”

Reply:
Thanks - I made the suggested change in #2734